src/lib/Liblog/pbs_log.c

```diff
@@ -386,22 +387,24 @@ log_add_debug_info()
 	/* To add leaf node name, if set */
 	strcpy(temp, "pbs_leaf_name=");
 	if(pbs_conf.pbs_leaf_name){
-		strncat(temp, pbs_conf.pbs_leaf_name, LOG_BUF_SIZE);
-		strncat(temp, ";", LOG_BUF_SIZE);
+		snprintf(dest, LOG_BUF_SIZE,"%s%s;",temp,pbs_conf.pbs_leaf_name);
+		strcpy(temp,dest);
 	}
 	else
 		strcat(temp, "N/A;");
-	strncat(tbuf, temp, LOG_BUF_SIZE);
-
+	snprintf(dest, LOG_BUF_SIZE,"%s%s",tbuf,temp);
+	strcpy(tbuf,dest);
 
 	/* To add mom node name, if set */
 	strcpy(temp, "pbs_mom_node_name=");
-	if(pbs_conf.pbs_mom_node_name)
-		strncat(temp, pbs_conf.pbs_mom_node_name, LOG_BUF_SIZE);
+	if(pbs_conf.pbs_mom_node_name){
```

Comment:
Please add a space before the open paren and open brace.

I think the code change is fine. Just need to be consistent with coding style.